projects/optic/src/commands/capture/actions/captureRequests.ts

```diff
@@ -139,13 +139,33 @@ function sendRequests(
   });
   return reqs.map(async (r) => {
     let verb = r.method || 'GET';
-    let opts = { method: verb };
+    let opts = {
+      method: verb,
+      headers: {},
+    };
+
+    if (r.data) opts['body'] = JSON.stringify(r.data);
+
+    try {
```

Comment:
I think you can remove this try block - I think this will only throw when `r.headers` isn't the shape we expect, and since we're running it through validation this shouldn't happen